src/js/profile/indicator.js

```diff
@@ -38,7 +38,15 @@ export class Indicator extends Observable {
             }
         }
 
-        let c = new Chart(this.formattingConfig, this.subindicators, this.groups, detail, 'Percentage', indicator, title);
+
+        const configuration = detail.indicators[title].chartConfiguration;
+        const config = {
```

Comment:
you don't need to create a new `config` variable here.
`Chart` is just using the `chartConfiguration` and not `formatting`
then you can refactor `Chart` again to just use `chartConfiguration`